src/pages/beheer/aanvragen/index.tsx

```diff
@@ -42,16 +43,41 @@ const CPRequestsPage: NextPage = () => {
 				className="p-cp-requests"
 				pageTitle={t('pages/beheer/aanvragen/index___aanvragen')}
 			>
-				<ScrollableTabs
-					tabs={statusFilters}
-					onClick={(tabId) =>
-						setFilters({
-							...filters,
-							status: tabId.toString(),
-						})
-					}
-					variants={['rounded', 'light', 'bordered']}
-				/>
+				<div className="p-cp-requests__header">
+					<ScrollableTabs
+						className="p-cp-requests__status-filter"
+						tabs={statusFilters}
+						variants={['rounded', 'light', 'bordered']}
+						onClick={(tabId) =>
+							setFilters({
+								...filters,
+								status: tabId.toString(),
+							})
+						}
+					/>
+
+					<SearchBar
+						backspaceRemovesValue={false}
+						className="p-cp-requests__search"
+						instanceId="requests-seach-bar"
+						light={true}
```

Comment:
FYI: you can omit the `={true}` if the value is true, since that is the default
```suggestion
						light
```